packages/common/constants/servers.ts

```diff
@@ -40,8 +44,8 @@ const SERVERS = {
 };
 
 const getDevelopmentEndpointUri = (packageName, { isDocker } = { isDocker: false }) =>
-  `http://${isDocker ? 'host.docker.internal' : SERVERS.DEVELOPMENT[packageName].host}:${
-    SERVERS.DEVELOPMENT[packageName].port
-  }`;
+  `http${SERVERS.DEVELOPMENT[packageName].https ? 's' : ''}://${
```

Comment:
Just a little tweak: `const { https, host, port } = SERVERS.DEVELOPMENT[packageName]` so there is no need to repeat the entire string. :-)